pipeline/plugins/adder/README.md

```diff
@@ -82,7 +82,14 @@ java -jar build/adder-all.jar [path to config]
 
 If you have a built version of this plugin, you can use the accompanied Dockerfile to set up an image with all pre-requisites installed. Please note that this Dockerfile will not compile the plugin for you.
```

Comment:
We should probably update this, as it now _does_ compile the plugin for you. Should probably also add a note that it assumes the build context is the root of the repository.